stl/inc/memory

```diff
@@ -308,10 +308,22 @@ _NoThrowFwdIt uninitialized_fill_n(_NoThrowFwdIt _First, const _Diff _Count_raw,
 }
 #endif // _HAS_IF_CONSTEXPR
 
+#if _HAS_CXX20
+template <class _Ty, class... _Types>
+_Ty* construct_at(_Ty* const _Location, _Types&&... _Args) noexcept(

```

Comment:
This doesn't implement WG21-N4849 [specialized.construct]/1: "*Constraints:* The expression `::new (declval<void*>()) T(declval<Args>()...)` is well-formed when treated as an unevaluated operand."